plugins/pmsmart/src/main/java/org/itsallcode/whiterabbit/plugin/pmsmart/PMSmartExporter.java

```diff
@@ -90,6 +120,22 @@ public void export(List<ProjectReportDay> daysToExport)
             weekViewPage.saveWeek();
         }
 
+        private void clearOtherProjects(final ProjectReportDay day)
+        {
+            final Iterator<Entry<String, ProjectRow>> it = projects.entrySet().iterator();
+            final Set<String> set = day.getProjects().stream()
+                    .map(project -> project.getProject().getCostCarrier())
+                    .collect(toSet());
+            while (it.hasNext() && !progressMonitor.isCanceled())
+            {
+                final Entry<String, ProjectRow> entry = it.next();
+                if (!set.contains(entry.getKey()))
+                {
+                    entry.getValue().enterDuration(day.getDate(), Duration.ZERO);
```

Comment:
Should we also delete the comment?

Reply:
Uh-oh.  Good question. I was shortly thinking about that before, but maybe forgot about it. Maybe you can guess my point? Speed, again. ☹️ 

1. To be perfectly correct WR indeed should delete the comment, yes.
2. On the other hand this would require so much overhead and additional runtime!
Skipping transfer of comments would heal this again, but do we want to force users into that? 
3. On the (third?) other hand: Who cares about comments for zero-duration activities? Maybe we can accept this?

So I came to the conclusion to live with comments for zero-duration activities and leave it like it is. 

What do you think?

Reply:
Good argument, that's fine for me. And you documented it in the readme :)